Infrastructure-Scripts/Wait-AzureDevOpsDeployment.ps1

```diff
@@ -0,0 +1,65 @@
+<#
+    .SYNOPSIS
+    Gets the latest deployment record for a given pipeline and environment and checks whether the current instance is able to run without clashing with said record.
+
+    .DESCRIPTION
+    Gets payload from Azure DevOps RestAPI and filters it to produce the build ID of the newest running build. Then compares to its own build ID, if equal to it then run and if not sleep and redo.
```

Comment:
Can the description include a link to the documentation of the Azure DevOps Rest API endpoint being used?